internal/component/otelcol/receiver/kafka/kafka.go

```diff
@@ -23,6 +26,7 @@ func init() {
 		Args:      Arguments{},
 
 		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			checkOutputSignals(args.(Arguments).Output, opts.Logger)
```

Comment:
I put a warn log here instead of returning an error with validate because it would be a breaking change for people who are using the workaround to have one receiver handling more than one signal. On the other hand it's an anti pattern so I would also be ok by making this an error